### PR TITLE
Add supported processors table

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
Fix front matter

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _ingest-pipelines/processors/index-processors.md [hunk 1]
```diff
@@ -23,4 +22,43 @@ GET /_nodes/ingest?filter_path=nodes.*.ingest.processors
...
+## Supported processors
+
+Processor types and their required or optional parameters vary depending on your specific use case. OpenSearch supports the following ingest processors. For tutorials on using these processors in an OpenSerch pipeline, go to each processor's respective documentation. 
+
+Processor type | Description
+:--- | :--- 
+`append` | Adds one or more values to a field in a document. 
+`bytes` | Converts a human-readable byte value to its value in bytes.
+`convert` | Changes the data type of a field in a document. 
+`csv` | Extracts CSVs and store them as individual fields in a document.
```
Comment (kolchfa-aws) [inline _ingest-pipelines/processors/index-processors.md:34]:
```suggestion
`csv` | Extracts CSVs and stores them as individual fields in a document. 
```

File: _ingest-pipelines/processors/index-processors.md [hunk 2]
```diff
@@ -23,4 +22,43 @@ GET /_nodes/ingest?filter_path=nodes.*.ingest.processors
...
+`geojson-feature` | Indexes GeoJSON data into a geospatial field.
+`grok` | Parses and structures unstructured data using pattern matching. 
+`gsub` | Replaces or deletes substrings within a string field of a document. 
+`html_strip` | Removes HTML tags from a text field and returns the plain text content. 
+`ip2geo` | Adds information about the geographical location of an IPv4 or IPv6 address.
+`join` | Concatenates each element of an array into a single string using a separator character between each element. 
+`json` | Converts a JSON string into a structured JSON object. 
+`kv` | Automatically parses key-value pairs in a field.
+`lowercase` | Converts text in a specific field to lowercase letters.
+`pipeline` | Is a series of configurable tasks called processors that run sequentially, making specific changes to incoming documents.
```
Comment (kolchfa-aws) [inline _ingest-pipelines/processors/index-processors.md:52]:
```suggestion
`pipeline` | Runs an inner pipeline.
```

**Review by kolchfa-aws — APPROVED** [review]
LGTM. A couple of small changes.

File: _ingest-pipelines/processors/index-processors.md [hunk 3]
```diff
@@ -23,4 +22,43 @@ GET /_nodes/ingest?filter_path=nodes.*.ingest.processors
...
+
+Processor types and their required or optional parameters vary depending on your specific use case. OpenSearch supports the following ingest processors. For tutorials on using these processors in an OpenSerch pipeline, go to each processor's respective documentation. 
+
+Processor type | Description
+:--- | :--- 
+`append` | Adds one or more values to a field in a document. 
+`bytes` | Converts a human-readable byte value to its value in bytes.
+`convert` | Changes the data type of a field in a document. 
+`csv` | Extracts CSVs and stores them as individual fields in a document. 
+`date` | Parses dates from fields and then uses the date or timestamp as the timestamp for the document.
```
Comment (natebower) [inline _ingest-pipelines/processors/index-processors.md:35]:
```suggestion
`date` | Parses dates from fields and then uses the date or timestamp as the timestamp for a document.
```

File: _ingest-pipelines/processors/index-processors.md [hunk 4]
```diff
@@ -23,4 +22,43 @@ GET /_nodes/ingest?filter_path=nodes.*.ingest.processors
...
+`date_index_name` | Indexes documents into time-based indexes based on a date or timestamp field in the document.
```
Comment (natebower) [inline _ingest-pipelines/processors/index-processors.md:36]:
```suggestion
`date_index_name` | Indexes documents into time-based indexes based on a date or timestamp field in a document. 
```

**Review by natebower — COMMENTED** [review]
@vagimeli Just two minor changes. Thanks!

**Pull request merged**